packages/@okta/vuepress-site/docs/reference/api/risk-events/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Risk Events
+category: management
+---
+
+# Risk Events API
+
+The Okta RiskEvents API provides the ability for RiskProviders to send RiskEvents to Okta.
```

Comment:
```suggestion
The Okta Risk Events API provides the ability for third-party Risk Providers to send Risk Events to Okta.
```

Reply:
fyi, as per style, updated to make separate words. Also added in "third-party", which I believe is correct, but highlighting here.

Regarding terminology: should we standardize on using "risk signals" or "risk events"? I generally see both used interchangeably.